pkg/dynamicaccess/act.go

```diff
@@ -0,0 +1,55 @@
+package dynamicaccess
+
+import (
+	"encoding/hex"
+
+	"github.com/ethersphere/bee/pkg/manifest"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type Act interface {
+	Add(lookupKey []byte, encryptedAccessKey []byte) Act
+	Get(lookupKey []byte) []byte
+	Load(lookupKey []byte) manifest.Entry
+	Store(me manifest.Entry)
```

Comment:
Load takes a `swarm address` as parameter, loads result to `self`, and gives back `error`.
Store does not have any input, and gives back `swarm_ref` and `error`.
// Viktor

Reply:
In progress: Ferenc

Reply:
PR: https://github.com/Solar-Punk-Ltd/bee/pull/15

Reply:
done